client/src/components/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
```

Comment:
Is the NavBar being bought in from the rebase?

Reply:
oui